pages.it/common/git-send-email.md

```diff
@@ -0,0 +1,25 @@
+# git send-email
+
+> Invia una raccolta di patch via email.
+> Le patch possono essere specificate come file, cartelle, o liste di revisione.
+> Maggiori informazioni: <https://git-scm.com/docs/git-send-email>.
+
+- Invia l'ultimo commit nel ramo corrente:
+
+`git send-email -1`
+
+- Invia un commit specifico:
+
+`git send-email -1 {{commit}}`
+
+- Invia 10 commit nel ramo corrente:
+
+`git send-email {{-10}}`
+
+- Invia un'email con un messaggio introduttivo alla serie di patch:
+
+`git send-email -{{numero di commit}} --compose`
```

Comment:
We use snake_case for variable names:
```suggestion
`git send-email -{{numero_di_commit}} --compose`
```
Looks an issue with the English page :disappointed:

Reply:
I could work on a bulk change, as there are other cases (e.g., buku, gist, man, ...)